File: lecshop_api_modules/lecshop_web_admin/src/main/webapp/res/js/order/ordersetting.js
// 基本路径
var basePath = $("#basePath").val();

var initDataTable = function () {
    LSFetch({
        url: basePath + '/queryordersetting.htm',
        success: function (data) {
            $('.summernote').summernote({
                height: 500,
                tabsize: 2,
                lang: 'zh-CN',
                onImageUpload: function (files, editor, $editable) {
                    sendFile(files, editor, $editable);
                }
            });
            $("#orderSettingId").val(data.id);
            $("#allowBack").val(data.allowBack);
            $("#aotuConfirm").val(data.aotuConfirm);
            $("#cashonDelivery").val(data.cashonDelivery);
            $("#returnDescs").code(data.returnDesc);
            $("#refunds").code(data.refundsDesc);
        }
    });
};

// 初始化列表
initDataTable();

// 刷新列表
var refreshDataTable = function () {
    $('#dataTable').DataTable().ajax.reload();
};

// 修改退货说明
function editReturn() {
    $("#return_dialog").modal("show");
}

// 保存退货说明
function saveReturn() {
    $("#return_dialog").modal("hide");
}

// 修改退款说明
function editRefunds() {
    $("#refund_dialog").modal("show");
}

// 保存退款说明
function saveRefunds() {
    $("#refund_dialog").modal("hide");
}

// 保存订单设置
function saveOrderSetting() {
    if (!$("#orderSettingForm").valid()) {
        return;
    }
    var orderSetting = {
        id: $("#orderSettingId").val(),
        allowBack: $("#allowBack").val(),
        aotuConfirm: $("#aotuConfirm").val(),
        cashonDelivery: $("#cashonDelivery").val(),
        returnDesc: $("#returnDescs").code(),
        refundsDesc: $("#refunds").code()
    };
    LSFetch({
        url: basePath + '/updateordersetting.htm',
        data: JSON.stringify(orderSetting),
        contentType: "application/json;charset=utf-8",
        success: function (res) {
            if (res == 1) {
                showerror('修改成功!');
            } else {
                showerror('修改失败');
            }
        }
    })
}

File: lecshop_base_modules/lecshop_order/src/main/java/com/lecshop/order/bean/OrderSetting.java
package com.lecshop.order.bean;

import lombok.Data;

/**
 * 订单设置实体类
 *
 * Created by LecShop on 2017/6/5.
 */
@Data
public class OrderSetting {

    /**
     * 主键id
     */
    private long id;

    /**
     * 是否可以退款和退货  0 可以 1 不可以 默认0
     */
    private String allowBack;

    /**
     * 订单自定收货几天前的订单 默认1
     */
    private int aotuConfirm;

    /**
     * 退款说明
     */
    private String refundsDesc;

    /**
     * 退货说明
     */
    private String returnDesc;

    /**
     * 是否支持货到付款  0 支付 1 不支持 默认0
     */
    private String cashonDelivery;

}
